spec/core_functions/color/color/relative_color.hrx

```diff
@@ -0,0 +1,88 @@
+<===> static/alpha/input.scss
+a {b: color(from #aaa srgb r g b / 25%)}
+
+<===> static/alpha/output.css
+a {
+  b: color(from #aaa srgb r g b/25%);
+}
+
+<===>
+================================================================================
+<===> static/no_alpha/input.scss
+a {b: color(from #aaa srgb r g b)}
+
+<===> static/no_alpha/output.css
+a {
+  b: color(from #aaa srgb r g b);
+}
+
+<===>
+================================================================================
+<===> calc/alpha/input.scss
+a {b: color(from #aaa srgb calc(l + 0.2) a b / 25%)}
+
+<===> calc/alpha/output.css
+a {
+  b: color(from #aaa srgb calc(l + 0.2) a b/25%);
```

Comment:
is `calc()` not evaluated to `1.2` because it's considered to be a string in a slash-separated list?

Reply:
`calc()` isn't evaluated to `1.2` because the first character is U+006C LATIN SMALL LETTER L and not U+0031 DIGIT ONE :slightly_smiling_face:.